packages/storage/src/indexed-db/index.ts

```diff
@@ -538,6 +540,25 @@ export class IndexedDb implements IndexedDbInterface {
     );
   }
 
+  async updatePrice(
+    pricedAsset: AssetId,
+    numeraire: AssetId,
+    numerairePerUnit: number,
+    height: bigint,
+  ) {
+    const estimatedPrice = new EstimatedPrice({
+      pricedAsset,
+      numeraire,
+      numerairePerUnit,
+      asOfHeight: height,
+    });
+
+    await this.u.update({
+      table: 'PRICES',
```

Comment:
This is missing the `key` right? or is it automatic?

Reply:
This works automatically if the key is specified via `keypath`